src/containers/error-boundary.jsx

```diff
@@ -28,21 +29,16 @@ class ErrorBoundary extends React.Component {
         window.history.back();
     }
 
+    handleReload () {
+        window.location.reload();
```

Comment:
Instead of calling this here, you can call `window.location.replace(window.location.origin)` to go back to the empty project url.